content/projekte/2023-04-01-fl-open-accident-map.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: project
+lab: [flensburg]
+imgname: flensburg/unfallkarte_stadt_flensburg.png
+title: Unfallorte auf einer interaktiven webbasierten Karte
+status: Sucht Mitstreiter
+
+links:
+- url: https://github.com/oklabflensburg/open-accident-map
+  name: GitHub
+- url: https://open-accident-map.de
+  name: Website
+
+collaborators:
+- name: Aurelius Wendelken
+  links:
+  - url: https://twitter.com/oklabflensburg
+    name: twitter
+  - url: https://github.com/oklabflensburg
+    name: github
+
+tags:
+- Geodaten
+- Unfallorte
+- Datenbank
+
+---
+
+Während einer kleinen Brainstorming Session ist diese interaktive webbasierte Karte mit den Daten des Bundesamtes für Kartographie und Geodäsie entstanden. Nach einigen Stunden der maschinellen Aufarbeitung und Betrachtung der Datensätze haben wir uns entschlossen, die Daten in eine Postgis Postgres Datenbank einzupflegen und zu indexieren. Unser Ziel ist es Entscheidungsträgern in Kommunen, Verwaltungen und auch privaten Unternehmen eine Datenbasierte Entscheidung anhand von Daten und Ereignissen zu treffen.
```

Comment:
```suggestion
Während einer kleinen Brainstorming Session ist diese interaktive webbasierte Karte mit den Daten des Bundesamtes für Kartographie und Geodäsie entstanden. Nach einigen Stunden der maschinellen Aufarbeitung und Betrachtung der Datensätze haben wir uns entschlossen, die Daten in eine PostGIS PostgreSQL Datenbank einzupflegen und zu indexieren. Unser Ziel ist es, Entscheidungsträger:innen in Kommunen, Verwaltungen und auch privaten Unternehmen zu ermöglichen, eine datenbasierte Entscheidung zu treffen.
```
PostGIS und PostgreSQL schreiben sich auf ihren eigenen Seiten so groß.

Dem letzten Satz fehlte irgendwie ein Ende

Wir haben angefangen, in neueren Texten vermehrt auf inklusive spräche, meist mit `:` zu achten.